### ISSUE TITLE
Letsencrypt renew-certs timeout 12.0.4

### ISSUE THREAD

**Issue by CorinthCom** [nextcloud/nextcloud-snap#433]
from log using sudo journalctl -u snap.nextcloud.renew-certs
renew-certs[23260]:    http://xxxxxx.xxxxxxxxx.co.uk/.well-known/acme-challenge/{token}:
renew-certs[23260]:    Timeout

https is enabled and the http request from Letsencrypt is re-directed to https

**Comment by CorinthCom** [comment]
Just to add, I've looked at [#401](https://github.com/nextcloud/nextcloud-snap/issues/401) & [#406](https://github.com/nextcloud/nextcloud-snap/issues/406)
A records are correct and working, I can access via HTTPS internal to network and external.

**Comment by kyrofa** [comment]
Yeah, you can try just running `nextcloud.enable-https lets-encrypt` again, no disable needed.

**Comment by CorinthCom** [comment]
Not entirely sure what was going on. I had switched off the port 80 redirection from the router and after re-enabling it auto-renewal failed several times (time-out).
I did disable and re-enable https and one of the certs renewed.
After then reading your reply above I re-ran the `nextcloud.enable-https lets-encrypt` and the other cert had already auto-renewed ??
In between steps I had used `apt-get update` and re-booted.

**Comment by kyrofa** [comment]
Huh. It's possible your domain records and IP addresses got out of sync there for a bit, but it's difficult to know for sure. Glad it worked itself out, though!
